Avoid version number in generated files.
Output will be identical between versions.
In case the output changes, build-compare will catch it.
Index: asciidoc-py3-9.0.0/html4.conf
===================================================================
--- asciidoc-py3-9.0.0.orig/html4.conf
+++ asciidoc-py3-9.0.0/html4.conf
@@ -460,7 +460,7 @@ endif::[]
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset={encoding}">
-<meta name="generator" content="AsciiDoc {asciidoc-version}">
+<meta name="generator" content="AsciiDoc">
 <meta name="description" content="{description}">
 <meta name="keywords" content="{keywords}">
 <title>{title}</title>
Index: asciidoc-py3-9.0.0/html5.conf
===================================================================
--- asciidoc-py3-9.0.0.orig/html5.conf
+++ asciidoc-py3-9.0.0/html5.conf
@@ -539,7 +539,7 @@ bodydata=<td class="tableblock halign-{c
 <html lang="{lang=en}">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset={encoding}">
-<meta name="generator" content="AsciiDoc {asciidoc-version}">
+<meta name="generator" content="AsciiDoc">
 <meta name="description" content="{description}">
 <meta name="keywords" content="{keywords}">
 <title>{title}</title>
Index: asciidoc-py3-9.0.0/slidy.conf
===================================================================
--- asciidoc-py3-9.0.0.orig/slidy.conf
+++ asciidoc-py3-9.0.0/slidy.conf
@@ -68,7 +68,7 @@ text=<p>|</p>
 <meta http-equiv="Content-Type" content="{quirks=application/xhtml+xml}{quirks?text/html}; charset={encoding}" />
 ifndef::copyright[<meta name="copyright" content="Copyright &#169; {author}" />]
 <meta name="copyright" content="Copyright &#169; {copyright}" />
-<meta name="generator" content="AsciiDoc {asciidoc-version}" />
+<meta name="generator" content="AsciiDoc" />
 <meta name="duration" content="{duration}" />
 ifdef::linkcss[]
 <link rel="stylesheet" href="{stylesdir=.}/{theme=asciidoc}.css" type="text/css" />
Index: asciidoc-py3-9.0.0/xhtml11.conf
===================================================================
--- asciidoc-py3-9.0.0.orig/xhtml11.conf
+++ asciidoc-py3-9.0.0/xhtml11.conf
@@ -531,7 +531,7 @@ cellspacing="0" cellpadding="4">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="{lang=en}">
 <head>
 <meta http-equiv="Content-Type" content="{quirks=application/xhtml+xml}{quirks?text/html}; charset={encoding}" />
-<meta name="generator" content="AsciiDoc {asciidoc-version}" />
+<meta name="generator" content="AsciiDoc" />
 <meta name="description" content="{description}" />
 <meta name="keywords" content="{keywords}" />
 <title>{title}</title>
